data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			label teeneepquestions
+			choice
+				`	"What will happen now then?"`
+					to display
+						not "Hai Reveal: what will happen now?"
+				`	"Do you think the Elders will listen?"`
+					goto listen
+					to display
+						not "Hai Reveal: will the elders listen?"
+				`	"Will there truly be a war between Hai and Unfettered?"`
+					goto war
+					to display
+						or
+							has "Hai Reveal: what will happen now?"
+							has "Hai Reveal: will the elders listen?"
+						not "Hai Reveal: will there be war?"
+				`	"What are the other options?"`
+					goto options
+					to display
+						has "Hai Reveal: will there be war?"
+				`	"Who is Aimet?"`
+					goto aimet
+					to display
+						has "Hai Reveal: will there be war?"
+			action
+				set "Hai Reveal: what will happen now?"
+			`	She shrugs. "It is honestly quite hard to say. I struggle to believe that the Elders could be so unwilling to act that they would risk escalating the standoff with the Unfettered into an actual war. Our brethren are no longer merely a disaffected rabble of veterans camping on the one world that supported radical change. They are a truly empowered movement with consolidated support at home and sympathizers amongst the Hai at large."`
+				goto teeneepquestions
+			label listen
+			action
+				set "Hai Reveal: will the elders listen?"
+			`	She sighs. "I wish to believe so, but part of me I can no longer ignore does not think it actually likely. Still, it is essential that I try with everything I have to make them do so. Otherwise there will be change, and we can be an unpredictable people in the face of that."`
+				goto teeneepquestions
+			label war
+			action
+				set "Hai Reveal: will there be war?"
+			`	She frowns and gently shakes her head. "No, or at least, not yet. The willingness of the Unfettered to threaten war is essential to all of the possible paths from here. Even if the Elders refuse to take action though, there are more steps between this meeting and that outcome. The options are not yet exhausted, though they are rapidly dwindling. With luck though, Aimet's influence might tip the situation in our favor."`
+				goto teeneepquestions
+			label options
+			`	"Ah, well those are difficult to explain. Since the options worth using will depend on exactly what happens in the meeting. One thing Sayari was very right to remind me off before is that there are normal channels and processes to follow, and those must be followed. We're an old people <last>, and not always a very patient people. We've forged a system of government with systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together."`
```

Comment:
```suggestion
			`	"Ah, well those are difficult to explain, since the options worth using will depend on exactly what happens in the meeting. One thing Sayari was very right to remind me of before is that there are normal channels and processes to follow, and those must be followed. We're an old people, Captain <last>, and not always a very patient people. We've forged a system of government with systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together."`
```